profiles/apparmor/apparmor.go

```diff
@@ -98,8 +98,10 @@ func InstallDefault(name string) error {
 	}
 	profilePath := f.Name()
 
-	defer f.Close()
-	defer os.Remove(profilePath)
```

Comment:
The order of these statements doesn't matter (at least on Linux). You can remove a file that is still open.